Jellyfin.Data/Entities/BaseItemMetadataField.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+
+namespace Jellyfin.Data.Entities;
+
+/// <summary>
+/// Enum MetadataFields.
+/// </summary>
+public class BaseItemMetadataField
+{
+    /// <summary>
+    /// Gets or Sets Numerical ID of this enumeratable.
+    /// </summary>
+    public required int Id { get; set; }
```

Comment:
I think we should either stick with Guid as PK everywhere or as int (and let be Guid just be public facing), but not mix both.

Reply:
This ID is an EnumValue `MetadataField` and does not reference any DB table